### PR TITLE
[DONTMERGE] net_4_5_xammac profile review.

### PR DESCRIPTION
**Pull request by chamons**

### CODE REVIEW

File: mcs/class/System.ServiceModel/net_4_5_xammac_System.ServiceModel.dll.sources [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+System.ServiceModel/EndpointIdentity.cs
+System.ServiceModel/IDuplexClientChannel.cs
+System.ServiceModel/OperationBehaviorAttribute.cs
+System.ServiceModel/RsaEndpointIdentity.cs
+System.ServiceModel/ServiceSecurityContext.cs
+System.ServiceModel/SpnEndpointIdentity.cs
+System.ServiceModel/UpnEndpointIdentity.cs
+System.ServiceModel/X509CertificateEndpointIdentity.cs
+Dummy_XM_4_5.cs
+../../../external/referencesource/System.ServiceModel.Internals/System/Runtime/UrlUtility.cs
```
Comment (marek-safar) [inline mcs/class/System.ServiceModel/net_4_5_xammac_System.ServiceModel.dll.sources]: Why do you need this when we deploy System.ServiceModel.Internals assembly?

File: mcs/Makefile [hunk 2]
```diff
@@ -14,6 +14,7 @@ mobile_SUBDIRS := build class
 mobile_static_SUBDIRS := build class
 binary_reference_assemblies_SUBDIRS := build class
 net_4_5_SUBDIRS := build mcs class nunit24 ilasm tools tests errors docs
+net_4_5_xammac_SUBDIRS := build class
```
Comment (marek-safar) [inline mcs/Makefile]: I'd prefer xammac_net_4_5 naming convention

File: mcs/build/profiles/net_4_5_xammac.make [hunk 3]
```diff
@@ -0,0 +1,10 @@
+include $(topdir)/build/profiles/net_4_5.make
```
Comment (marek-safar) [inline mcs/build/profiles/net_4_5_xammac.make]: I'd prefer xammac_net_4_5 naming convention

File: mcs/class/Facades/Makefile [hunk 4]
```diff
@@ -18,6 +18,7 @@ net_4_5_SUBDIRS = $(monotouch_SUBDIRS) System.Reflection.Emit.ILGeneration Syste
 monodroid_SUBDIRS = $(net_4_5_SUBDIRS)
 
 xammac_SUBDIRS = $(net_4_5_SUBDIRS)
+net_4_5_xammac_SUBDIRS = $(net_4_5_SUBDIRS)
```
Comment (marek-safar) [inline mcs/class/Facades/Makefile]: I'd prefer xammac_net_4_5 naming convention

File: mcs/class/System.ServiceModel/net_4_5_xammac_System.ServiceModel.dll.sources (hunk 1 above)
Comment (chamons) [inline mcs/class/System.ServiceModel/net_4_5_xammac_System.ServiceModel.dll.sources]: Fixed via: d0610848aa2955993accc786d2b397efc405ffff

File: mcs/build/profiles/net_4_5_xammac.make (hunk 3 above)
Comment (chamons) [inline mcs/build/profiles/net_4_5_xammac.make]: Fixed with d120b55cb3b5c3e887afa872544a189ed189e654

File: mcs/class/Facades/Makefile (hunk 4 above)
Comment (chamons) [inline mcs/class/Facades/Makefile]: Fixed with d120b55cb3b5c3e887afa872544a189ed189e654

**Pull request closed without merging**